geometry_msgs/Vector3 min_bound
geometry_msgs/Vector3 max_bound
geometry_msgs/Vector3 fov
---
trajectory_msgs/MultiDOFJointTrajectory trajectory
